src/main/java/com/conveyal/gtfs/loader/JdbcGtfsExporter.java

```diff
@@ -124,26 +127,70 @@ public FeedLoadResult exportTables() {
             result.fareAttributes = export(Table.FARE_ATTRIBUTES);
             result.fareRules = export(Table.FARE_RULES);
             result.feedInfo = export(Table.FEED_INFO);
-            // FIXME: only write frequencies for "approved" routes using COPY TO with results of select query
-            result.frequencies = export(Table.FREQUENCIES);
-            // FIXME: only write "approved" routes using COPY TO with results of select query
-            result.routes = export(Table.ROUTES);
+            // Only write frequencies for "approved" routes using COPY TO with results of select query
+            String frequencySelectSql = null;
+            if (fromEditor) {
+                // Generate filter SQL for trips if exporting from the editor.
+                // The filter clause for frequencies requires two joins to reach the routes table and a where filter on
+                // route status.
+                frequencySelectSql = Table.FREQUENCIES.generateSelectSql(feedIdToExport, Requirement.OPTIONAL) +
```

Comment:
Rather than concatenating the strings with the + operator it might make more sense from a reader's point of view to pass them all to join() with spaces as the separator.

Reply:
Also then you don't have to strategically inject spaces at the beginning and end of the SQL fragments generated by your methods.